CONTRIBUTING.rst

```diff
@@ -0,0 +1,234 @@
+=================
+How to contribute
+=================
+
+First, thanks for considering contributing to Astrality, that means a lot!
+Here we describe how you can help out, either by improving the documentation, submitting issues, or commit pull requests.
+
+If you end up contributing, please consider adding yourself to the file ``CONTRIBUTORS.rst``.
+
+.. _contributing_issues:
+
+Bug reports and feature requests
+================================
+
+Bug reports and feature requests can be submitted in the form of a `GitHub issue <https://github.com/JakobGM/astrality/issues/new>`_.
```

Comment:
Maybe just link to /issues and mention to first check existing open/closed issues to see if a similar topic has been discussed before

Reply:
Done!